File: api/Entities/Admin/Order/OrderItem.cs
using api.Entities.Master;

namespace api.Entities.Admin.Order
{
    public class OrderItem: BaseEntity
    {
        public int OrderId { get; set; }
        public int SrNo { get; set; }
        public int ProfessionId { get; set; }
        public string SourceFrom { get; set; }
        public int Quantity { get; set; }
        public int MinCVs { get; set; }
        public int MaxCVs { get; set; }
        public bool Ecnr { get; set; }=false;
        public DateOnly CompleteBefore { get; set; }
        public int Charges { get; set; }
        public string Status { get; set; }="Not Started";
        public bool Checked {get; set;}
        public string ReviewItemStatus { get; set; }="NotReviewed";
        public ICollection<OrderItemAssessmentQ> OrderItemAssessmentQs { get; set; }
        public Profession Profession { get; set; }
        public JobDescription JobDescription { get; set; }
        public Remuneration Remuneration { get; set; }
        public ContractReviewItem ContractReviewItem {get; set;}
    }
}

File: client/src/app/_models/admin/orderItem.ts
import { IContractReviewItem } from "./contractReviewItem";
import { IJobDescription } from "./jobDescription";
import { IRemuneration } from "./remuneration";

export interface IOrderItem{
     selected: boolean;
     id: number; 
     orderId: number;
     orderNo: number; 
     srNo: number; 
     professionId: number;
     ecnr: boolean; 
     sourceFrom: string; 
     quantity: number; 
     minCVs: number; 
     maxCVs: number;
     //requireInternalReview: boolean; 
     //requireAssess: boolean; 
     completeBefore: Date; 
     //assignedId: number; 
     //assignedToName: string; 
     //charges: number; 
     //feeFromClientINR: number; 
     status: string;
     reviewItemStatus: string;
     checked: boolean;
     jobDescription: IJobDescription; 
     remuneration: IRemuneration;
     contractReviewItem: IContractReviewItem;
}